README.md

```diff
@@ -42,6 +43,14 @@ before running them.
 Version History
 ---------------
 
+0.2.11 (22/02/2020)
+
+*   Added method to create a user by admin and set the status to CONFIRMED - [abhi36](https://github.com/abhi36)
+
+0.2.11 (06/01/2020)
+
+Added method to get a user by an access token - bjoernHeneka
+
```

Comment:
Can you remove these changes? if this PR is merged, I'll need to update the date and version anyway so I'll add make these changes myself.

Reply:
Updated

Reply:
The changes are still here in the diff.

Reply:
Just a reminder about this one.

Reply:
This change is still here.

Reply:
I updated it after interactive rebase to my branch. Not sure how it's still not getting updated. Those lines are even not showing in my codebase now

Reply:
It looks like it's on this commit https://github.com/pmill/aws-cognito/pull/39/commits/745e545535e7fd468fbee4810d7b5cceb71c5331